pkg/sql/parser/builtins.go

```diff
@@ -1442,18 +1442,22 @@ var Builtins = map[string][]Builtin{
 	// and the session's database search path.
 	"current_schemas": {
 		Builtin{
-			Types:        ArgTypes{{"unused", TypeBool}},
```

Comment:
huh that's because pg specifies that this function takes one argument. Even though CockroachDB doesn't give it any meaning we should still accept the argument for compatibility.